templates/shared/pricing/_openstack-packages.html

```diff
@@ -60,4 +61,16 @@ <h2 class="p-heading--four">Fully managed cloud</h2>
     <p><a href="/openstack/managed">Fully Managed OpenStack&nbsp;&rsaquo;</a></p>
   </div>
   {% endif %}
+  <div class="u-no-fixed-width">
+    <p id="os-footnote">
+      <small>
+      <sup><a href="#os-footnote">*</a></sup> Additional features, functionality and integrations are available via add-ons
+      </small>
+    </p>
+      <p class="p-button">
+        <a href="https://assets.ubuntu.com/v1/d4766546-Private-Cloud-Build_Datasheet.pdf">
```

Comment:
```suggestion
        <a href="https://assets.ubuntu.com/v1/d4766546-Private-Cloud-Build_Datasheet.pdf" class="p-link--external">
```

Reply:
We don't do this on assets.

Reply:
I would have thought it was important to show user they are about to leave the site and be shown a PDF?